packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -1075,7 +1075,7 @@ describe('Dropdown', () => {
 
       expect(wrapper.find(Dropdown).state('searchQuery')).toBe('foo')
 
-      wrapper.setProps({ children: <Dropdown items={items} search searchQuery="bar" /> })
```

Comment:
This update also eliminates this craziness: previously we should update whole tree...